packages/http-client/src/http-client.spec.ts

```diff
@@ -494,34 +535,44 @@ sap-client:001`);
       requestSpy.mockRestore();
     });
 
-    it('uses the default timeout if not given', async () => {
-      const destination: Destination = {
-        url: 'https://destinationUrl'
-      };
-      const requestSpy = jest.spyOn(axios, 'request').mockResolvedValue(true);
+    it('uses no timeout if no timeout middleware given', async () => {
+      const delayInResponse = 10000 + 1; // typical default timeouts of http clients are 10 seconds
+      nock('https://example.com', {})
+        .get('/with-delay')
+        .delay(delayInResponse)
+        .reply(200);
```

Comment:
```suggestion
      nock('https://example.com', {})
        .get('/with-delay')
        .delay(10001) // typical default timeouts of http clients are 10 seconds
        .reply(200);
```
[pp] The variable is not really useful in my opinion. The name does not information and it is only used once.